### PR TITLE
ENCD-5462-qc-grouping

### PR DESCRIPTION
**Pull request by yunhailuo**

### CODE REVIEW

File: src/encoded/static/scss/encoded/modules/_pipeline.scss [hunk 1]
```diff
@@ -245,6 +245,10 @@ g.pipeline-node-qc-metric {
         @media screen and (min-width: $screen-md-min) {
             flex: 0 1 33.33%;
         }
+
+        hr {
+            margin-right: 10px;
+        }
```
Comment (forresttanaka) [inline src/encoded/static/scss/encoded/modules/_pipeline.scss]: Could you move this hr rule into the @media above? Right not there’s too much margin on the right on mobile, and putting it in that @media would disable this rule on mobile.

**Review by forresttanaka — CHANGES_REQUESTED** [review]
Just need to move the hr rule.

File: src/encoded/static/scss/encoded/modules/_pipeline.scss (hunk 1 above)
Comment (yunhailuo) [reply src/encoded/static/scss/encoded/modules/_pipeline.scss]: Thanks a lot for checking that. Missed mobile completely.

File: src/encoded/static/scss/encoded/modules/_pipeline.scss [hunk 2]
```diff
@@ -244,7 +244,11 @@ g.pipeline-node-qc-metric {
     @at-root #{&}__data {
         @media screen and (min-width: $screen-md-min) {
             flex: 0 1 33.33%;
+            hr {
+                margin-right: 10px;
```
Comment (forresttanaka) [inline src/encoded/static/scss/encoded/modules/_pipeline.scss]: Sorry, I guided you wrong on this one. SCSS nesting can get complicated at times, as in this case where the outer nested `.quality-metrics-modal` gets ignored, so now the hr is too narrow on the file page. I tested a change that works: move the hr out of here and into the @media for `.quality-metrics-modal`, just under the `display: flex`.

**Review by forresttanaka — CHANGES_REQUESTED** [review]
Sorry, I led you the wrong way.

File: src/encoded/static/scss/encoded/modules/_pipeline.scss (hunk 2 above)
Comment (yunhailuo) [reply src/encoded/static/scss/encoded/modules/_pipeline.scss]: Thank you very much! There is no way I could figure this out myself.

**Review by forresttanaka — APPROVED** [review]
Great!

**Review by gabdank — APPROVED** [review]

**Pull request merged**